plugin-build/src/main/kotlin/io/sentry/android/gradle/sourcecontext/BundleSourcesTask.kt

```diff
@@ -25,6 +26,7 @@ import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
 import org.gradle.api.tasks.TaskProvider
 
+@CacheableTask
```

Comment:
I was under the impression that Exec tasks can't be really cacheable, but looking at the inputs and outputs it makes sense! Thanks for fixing!

Reply:
Yeah, it looks like it makes sense to cache here but of course you might know better than me if there are untracked inputs/outputs here.